index.js

```diff
@@ -20,8 +21,30 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    Recipe.create(data, (err, recipe) => {
+      recipe.forEach(Element => {
+        console.log(Element.title)
+      })
+    Recipe.updateOne({title: "Rigatoni alla Genovese"}, {duration: 100})
+    Recipe.deleteOne({title: "Carrot Cake"})
+      .then(() => {console.log("updated")})
+      .catch(() => {console.log("not updated")})
+    })
```

Comment:
You can chain your promises and make use of the returned Promise values in the `.then()` that follows after

```suggestion
    Recipe.create(recipe)
        .then((results) => {
          console.log(`Title of the recipe: ${results}`
          
          return Recipe.insertMany(data);
         }))
         .then((savedRecipes) => {
           console.log(`All recipes saved in the database: ${savedRecipes}`
           
           return Recipe.find({});
         })
         .then((allRecipesFromDB) => {
           console.log(`All recipes in the database: ${allRecipesFromDB}`
           
           return Recipe.deleteOne({ title: "Carrot Cake" });
         }).then((deletedResults) => {
           console.log(`Recipe deleted from database: ${deletedResults}`;
           
           return Recipe.findOneAndUpdate({ title:"Rigatoni alla Genovese"}, {duration : 100});
         })
         .then((updatedResults) => {
           console.log(`Recipe updated: ${updatedResults}`;
         })
```